bevy-jornet/examples/whac-a-square.rs

```diff
@@ -252,7 +359,10 @@ mod menu {
         }
     }
 
-    fn despawn_menu(mut commands: Commands, root_ui: Query<Entity, (With<Node>, Without<Parent>)>) {
+    fn despawn_menu(
+        mut commands: Commands,
+        root_ui: Query<Entity, (With<Node>, With<MenuUI>, Without<Parent>)>,
```

Comment:
I added more control on what exactly is despawned to be able to keep the debug UI for errors display